### DISCUSSION TITLE
Sublime Text Build 4152 broke the Dracula colors for Python

### DISCUSSION THREAD

**Discussion by alcala21** [dracula/dracula-theme#883] [General] [answered]
As the title says, I updated Sublime Text to the latest build 4152, and it broke the Dracula color scheme for Python. All objects are now blue and have a line under them. Can you look into it?

Thank you.

**Comment by alnah** [comment]
Same issue here. It makes the code pretty non-readable. I'm on Dracula Pro.

**Reply by alnah** [reply]
Ajude a gente por favor Zeno! :-)

**Comment by luxonauta** [comment, marked as answer]
I will be investigating soon. Thanks for reporting!
